mt_sat/mt_sat_wrapper.m

```diff
@@ -98,39 +119,42 @@ function mt_sat_wrapper(mtw_nii,pdw_nii,t1w_nii,mtw_jsn,pdw_jsn,t1w_jsn,varargin
     qMRLabVer;
 end
 
-Model = mt_sat; 
+% ==== Set Protocol ====
+Model = mt_sat;
 data = struct();
 
+% Load data
+data.MTw=double(load_nii_data(mtw_nii));
+data.PDw=double(load_nii_data(pdw_nii));
+data.T1w=double(load_nii_data(t1w_nii));
+
+
+
+
+
 customFlag = 0;
-if all([isempty(mtw_jsn) isempty(pdw_jsn) isempty(t1w_jsn)]); customFlag = 1; end; 
+if all([isempty(mtw_jsn) isempty(pdw_jsn) isempty(t1w_jsn)]); customFlag = 1; end;
 
-% Account for optional inputs and options.
+%Account for optional inputs and options
 if nargin>6
```

Comment:
You'll need to edit this block as well.

Reply:
For the remaining optional arguments, I also get the values using an `if` statement.